docs/supported/flockercontainers.rst

```diff
@@ -8,20 +8,16 @@ Running Flocker in Containers
 
     <div class="admonition labs">
         <p>This page describes one of our experimental projects, developed to less rigorous quality and testing standards than the mainline Flocker distribution. It is not built with production-readiness in mind.</p>
```

Comment:
This isnt part of our release process to update the containers, but I feel could be integrated. E.g. this documentation is probably out of date in the sense that the containers for 1.14 , 1.15 havent been built I think, i usually kick these off, last I did was fro christian at jetstack around 1.13 I think.